@xen-orchestra/web/src/components/backups/configuration/BackupJobSchedules.vue

```diff
@@ -0,0 +1,226 @@
+<template>
+  <UiCard>
+    <UiTitle> {{ t('schedules') }} </UiTitle>
+    <UiQuerySearchBar @search="value => (searchQuery = value)" />
+    <UiTopBottomTable :selected-items="0" :total-items="0">
+      <UiTablePagination v-bind="paginationBindings" />
+    </UiTopBottomTable>
```

Comment:
```suggestion
  <UiCard class="backup-jobs-schedules">
    <UiTitle> {{ t('schedules') }} </UiTitle>
    <div class="container">
      <div class="table-actions">
        <UiQuerySearchBar @search="value => (searchQuery = value)" />
        <UiTopBottomTable :selected-items="0" :total-items="0">
          <UiTablePagination v-bind="paginationBindings" />
        </UiTopBottomTable>
      </div>
```

I think you can use same structure as other table, like `backupJobsTable`